### ISSUE TITLE
BattleScribe 2.0

### ISSUE THREAD

**Issue by Jonskichov** [BSData/star-wars-armada#18]
Hi guys.

BattleScribe 2.0 is coming and it contains some major changes to Data Editor and the data format.

While it should happily upgrade 1.15.x format data files to the new format, you never know what sneaky bugs might be lurking...

It would help a lot if you could run your data through it and make sure nothing gets broken so we have no nasty surprises on release day (whenever that might be...).

**Alpha Downloads:**
Desktop: https://github.com/BattleScribe/Desktop-Alphas/issues/1
Android: https://github.com/BattleScribe/Android-Alphas/issues/1
iOS: https://github.com/BattleScribe/iOS-Alphas/issues/1

If you find any issues, please do let me know: https://github.com/BattleScribe/Desktop-Alphas/issues

**Comment by BrianisBlack** [comment]
Committing to iOS testing for the star-wars-armada files.

**Comment by XWingDrLucky** [comment]
I cleaned up quite a bit of the catalogue junk that got added during the automatic file conversion to 2.0.  I also centralized a few things to the gst file (mostly Objectives and Rules).  As of right now, I think everything is happy and compatible with the new version, but there's quite a bit of functionality in BS2.0 we aren't taking advantage of.  If we centralized a ton of data to the gst, basically most of the upgrades and maybe units, we could do some cool stuff with multiple forces (like something for the grand finale battle of Corellian Conflict).  However, that's pretty much a full rebuild of the catalogues and would blow up all lists, so it might now be worth it...
